package.json

```diff
@@ -20,6 +20,7 @@
     "babel-preset-react": "^6.11.1",
     "bower": "^1.8.8",
     "bower-resolve-webpack-plugin": "^1.0.3",
+    "check-engine": "^1.10.1",
```

Comment:
**Question**; don't you have to add the package.lock.json too ?

Reply:
Becuase is ignored in .gitignore